packages/experiments/src/theming/examples/ThemeDialog.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import { DefaultButton, PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import { ChoiceGroup } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { Dialog, DialogType, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+
+export interface IDialogExampleProps {
+  buttonText: string;
+}
+
+export interface IDialogExampleState {
+  hideDialog: boolean;
+}
+
+export class ThemeDialog extends React.Component<IDialogExampleProps, IDialogExampleState> {
```

Comment:
nit: this is just a normal dialog with no theming-specific code, so it shouldn't necessarily be called ThemeDialog
we could even re-use an existing dialog example

Reply:
Yeah, this has been refactored a few times while developing this feature so the naming is obsolete. At this point it looks no different from the dialog example, so I'll clean it up.